native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  View,
+  Platform,
+  StyleSheet,
+} from 'react-native';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const {
+  spacingBase,
+  colorGray900,
+  colorWhite,
+} = tokens;
+
+/**
+ * Define styles needed by the component
+ */
+const styles = StyleSheet.create({
+  card: {
+    // TODO: update once tokens are available
+    borderRadius: 8,
+    shadowColor: colorGray900,
+    shadowOffset: { width: 0, height: 1 },
+    shadowOpacity: 0.3,
+    shadowRadius: 3,
+    backgroundColor: colorWhite,
+  },
+});
+
+const BpkCard = (props) => {
+  const { padded } = props;
+  const paddingStyle = { padding: spacingBase };
```

Comment:
We should err on the side of using `Stylesheet.create` so can't we do this by adding 

```
padded: {
  padding: spacingBase,
}
```

when creating the styles and conditionally applying it?

Reply:
absolutely right, the previous revision had `nopadding` as prop rather than `padded` now what you're suggesting makes more sense, I'll amend